cli/js/web/abort_controller.ts

```diff
@@ -3,14 +3,10 @@
 import { AbortSignalImpl, signalAbort } from "./abort_signal.ts";
 
 export class AbortControllerImpl implements AbortController {
-  #signal = new AbortSignalImpl();
-
-  get signal(): AbortSignal {
-    return this.#signal;
-  }
+  readonly signal = new AbortSignalImpl();
 
   abort(): void {
-    this.#signal[signalAbort]();
+    this.signal[signalAbort]();
```

Comment:
This changes the behavior, now it's possible to do:

```
const signal = new AbortController();
signal.signal = 5;
signal.signal === 5; // true
```

Reply:
Yes, this is against the spec, why make this change?

Reply:
> Yes, this is against the spec, why make this change?

My bad, i'll revert it